configs/components/echam/echam.yaml

```diff
@@ -529,6 +497,24 @@ choose_wiso_code:
                                 add_coupling_fields:
                                         "[[wiso_fields-->FIELD]]":
                                                 grid: atmo
+                                add_streams:
+                                        - wiso
+                                        - accw_wiso
+                                        - sf_wiso
+                                        - rad
+                                add_forcing_files:
+                                        ozonea: piozonea
+                                        ozoneb: piozoneb
+                                        ozonec: piozonec
+                                add_forcing_in_work:
+                                        ozonea: "ozon1988"
+                                        ozoneb: "ozon1989"
+                                        ozonec: "ozon1990"
+                                restart_in_sources:
+                                        "[[streams-->STREAM]]": restart_${parent_expid}_${parent_date!syear!smonth!sday}_STREAM.nc
+                                restart_out_sources:
+                                        "[[streams-->STREAM]]": restart_${general.expid}_${other_date!syear!smonth!sday}_STREAM.nc
+
```

Comment:
@xshi-awi , @mwerner-awi I moved all this from the `choose_version.6.3.05p2-wiso` to be under the `with_wiso` switch. Is that correct? (i.e. a peson is running echam with wiso code but don't want WISO in their simulation, then all these variables here are not needed for that)

Reply:
This is the whole block for which I would like to have your feedback:
```
                choose_with_wiso:
                        True:
                                nwiso: 3
                                lwiso_rerun: ${lresume}
                                add_coupling_fields:
                                        "[[wiso_fields-->FIELD]]":
                                                grid: atmo
                                add_streams:
                                        - wiso
                                        - accw_wiso
                                        - sf_wiso
                                        - rad
                                add_forcing_files:
                                        ozonea: piozonea
                                        ozoneb: piozoneb
                                        ozonec: piozonec
                                add_forcing_in_work:
                                        ozonea: "ozon1988"
                                        ozoneb: "ozon1989"
                                        ozonec: "ozon1990"
                                restart_in_sources:
                                        "[[streams-->STREAM]]": restart_${parent_expid}_${parent_date!syear!smonth!sday}_STREAM.nc
                                restart_out_sources:
                                        "[[streams-->STREAM]]": restart_${general.expid}_${other_date!syear!smonth!sday}_STREAM.nc
```

Reply:
I think the `restart_in_sources` and `restart_out_sources` can be outside of the `choose_with_wiso` block. The only important part (as far as I know) is the add strap and add coupling field part.

Reply:
you are right about the `retart_in_sources` and `restart_out_sources`, it's all happening from the `add_stream`. What about the `forcing_files` then? is this something `with_wiso` specific?